src/Agent/NewRelic/Agent/Extensions/Providers/Wrapper/MassTransit/MassTransitHelpers.cs

```diff
@@ -17,18 +17,25 @@ private static string[] GetQueueData(Uri sourceAddress)
             return items[items.Length - 1].Split('?');
```

Comment:
Do we need to check `items.Length > 0` here? Is it possible for `sourceAddress` to not contain any underscore characters?

Reply:
If `sourceAddress` has no underscores, `Split` will return a list with a single element containing the whole input string.  So, `items.Length` should always be at least 1, and in that case, we will be calling `items[0].Split('?')`.  So, this shouldn't cause any exceptions to be thrown.  However, the queue name could end up being bogus in that case.  I'll put in some more checks and return with sensible defaults.